src/build.ts

```diff
@@ -22,7 +24,12 @@ export async function build(config: BuildConfig): Promise<void> {
   const start = Date.now();
 
   const pkgDir = normalizePath(config.cwd);
-  const pkg = await readJSON(join(pkgDir, "package.json")).catch(() => ({}));
+  const jiti = createJiti(pkgDir);
+  const pkg: PackageJson =
+    ((await jiti.import("./package.json", {
```

Comment:
Why do we need `jiti` here?
`pkg-types` has a `readPackageJSON()` method for this.